chromium/store.js

```diff
@@ -9,14 +9,18 @@ function setStorage(store) {
   });
 }
 
-setStorage(chrome.storage.local);
 if (chrome.storage.sync) {
   chrome.storage.sync.set({"sync-set-test": true}, () => {
-    if(!chrome.runtime.lastError){
+    if(chrome.runtime.lastError){
+      setStorage(chrome.storage.local);
+    } else {
       setStorage(chrome.storage.sync);
-      background.initializeStoredGlobals();
     }
+    background.initialize();
   });
+} else {
+  setStorage(chrome.storage.local);
+  background.initialize();
```

Comment:
Instead of calling `background.initialize` here, could export an initialization function for this module. Then in `background.initialize` call `store.initialize`?

Reply:
Since `background.js` is the last thing to load, I think it makes more sense to orchestrate the initialization there.